### ISSUE TITLE
Skipping cells during execution

### ISSUE THREAD

**Issue by amardeep** [ploomber/ploomber-engine#37]
It would be good to have a way to skip certain cells during execution of jupyter notebooks.

My use case is that it is useful to have extra cells which might be useful for a developer while debugging or during interactive development - but shouldn't be run in regular runs.

But there are other scenarios discussed in the thread here:
https://github.com/nteract/papermill/issues/429

**Comment by edublancas** [comment]
I quickly skimmed through papermill's issue. looks like the community is in favor of a `remove` feature instead of a `skip` one since skipping will create non-linearly executed notebooks.

I think we can start with a simple implementation and improve it with community feedback: we can have the executor remove cells tagged with a specific tag and then execute the notebook. Would this work for you?

Something like:

```sh
ploomber-engine input.ipynb output.ipynb --remove-tagged-cells=some-tag
```

**Comment by idomic** [comment]
Maybe worth making it static so you don't need to pass it as a var each time..

`ploomber-engine input.ipynb output.ipynb`

And in `input.ipynb` mark those cells with a remove tag.

**Comment by edublancas** [comment]
> Maybe worth making it static so you don't need to pass it as a var each time..

I thought the same at the beginning, but since this will affect the execution flow, I think it's better to be explicit.

**Comment by idomic** [comment]
We should do it explicitly, and then make sure we support an external file or an env variable so users don't have to repeat those multiple times.

**Comment by edublancas** [comment]
> external file or an env variable 

this is more work than just the remove feature so I think we should split in two issues. this one should only cover the skipping part.

**Comment by idomic** [comment]
Yeah, I'll reference a new issue.

**Comment by amardeep** [comment]
> I quickly skimmed through papermill's issue. looks like the community is in favor of a `remove` feature instead of a `skip` one since skipping will create non-linearly executed notebooks.
> 
> I think we can start with a simple implementation and improve it with community feedback: we can have the executor remove cells tagged with a specific tag and then execute the notebook. Would this work for you?
> 
> Something like:
> 
> ```shell
> ploomber-engine input.ipynb output.ipynb --remove-tagged-cells some-tag
> ```

Yes, that works. Thank you.

**Comment by edublancas** [comment]
released in version 0.0.21!
